src/main/java/frc/robot/utils/diag/DiagClimber.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.utils.diag;
+
+import com.revrobotics.CANSparkMax;
+
+public class DiagClimber extends DiagSparkMaxEncoder {
```

Comment:
why make a diag that does the same thing as the original? you can just change the title when you add it to shuffleboard